documentation/docs/migration-guide.md

```diff
@@ -111,6 +111,31 @@ This example could easily have been migrated using `ByNodePropertyNodeSet`, but
 other cases this might not be as straight forward as shown.
 :::
 
+#### A note on `applyToAllChildren` to style all descendants of a node
+
+In previous versions, all styling functions accepted an optional `applyToAllChildren`-argument.
+When this was `true`, Reveal would apply the styling to all descendants. When migrating
+to the new version, adding the root tree index of the subtree to `ByTreeIndexNodeSet` will not have
+the same result. Instead, use `Cognite3DModel.getSubtreeIndices()` to get the full range of tree
```

Comment:
should subtreeSize be mentioned here as a valid alternative?

Reply:
Good point - added